### ISSUE TITLE
Failed to run case on M1 mac (arm/v7 platform)

### ISSUE THREAD

**Issue by evalieungh** [NorESMhub/noresm-land-sites-platform#104]
@MichalTorma posted [this issue](https://github.com/MetOs-UiO/fates_platform_ecological_climatology/issues/3) on the MetOS Ecological Climatology repo. Probably relevant for the main platform too.

**Comment by ka7eh** [comment]
Does this issue still exist with the latest changes? Can you share the content of the error log?
For the example in the above-mentioned issue, it should be in `resources/cases/0da25a1c6dcbde3ff07b9b5fb921d1df/bld/lnd.bldlog.220511-093703`.

**Comment by evalieungh** [comment]
@MichalTorma if you have time, do you want to test again?

**Comment by MichalTorma** [comment]
Hi there, sorry for late reply - I was at the conference :) . Anyway now my issue is that when I try to run `docker compose up` I get 
```zsh
WARN[0000] The "HOST_USER" variable is not set. Defaulting to a blank string.
WARN[0000] The "HOST_UID" variable is not set. Defaulting to a blank string.
WARN[0000] The "HOST_USER" variable is not set. Defaulting to a blank string.
WARN[0000] The "HOST_UID" variable is not set. Defaulting to a blank string.
[+] Running 0/1
 ⠿ ui Error                                                                                                                                                                1.3s
Error response from daemon: Head "https://ghcr.io/v2/noresmhub/fates-platform/manifests/latest": denied
```
Do you have the repository set to public?

**Comment by ka7eh** [comment]
The repo is public, but its name has been changed. Have you pulled the latest version? You might have to update your repo remote too, if it's out-of-date. Try with the main branch from https://github.com/NorESMhub/noresm-land-sites-platform.

**Comment by MichalTorma** [comment]
New error:
```zsh
#0 33.83 Building wheels for collected packages: cftime
#0 33.83   Building wheel for cftime (pyproject.toml): started
#0 34.25   Building wheel for cftime (pyproject.toml): finished with status 'error'
#0 34.26   error: subprocess-exited-with-error
#0 34.26   
#0 34.26   × Building wheel for cftime (pyproject.toml) did not run successfully.
#0 34.26   │ exit code: 1
#0 34.26   ╰─> [18 lines of output]
#0 34.26       running bdist_wheel
#0 34.26       running build
#0 34.26       running build_py
#0 34.26       creating build
#0 34.26       creating build/lib.linux-aarch64-cpython-310
#0 34.26       creating build/lib.linux-aarch64-cpython-310/cftime
#0 34.26       copying src/cftime/__init__.py -> build/lib.linux-aarch64-cpython-310/cftime
#0 34.26       copying src/cftime/_strptime.py -> build/lib.linux-aarch64-cpython-310/cftime
#0 34.26       running build_ext
#0 34.26       building 'cftime._cftime' extension
#0 34.26       creating build/temp.linux-aarch64-cpython-310
#0 34.26       creating build/temp.linux-aarch64-cpython-310/tmp
#0 34.26       creating build/temp.linux-aarch64-cpython-310/tmp/pip-install-6klh6n1t
#0 34.26       creating build/temp.linux-aarch64-cpython-310/tmp/pip-install-6klh6n1t/cftime_bd2c91e5ac164b95a6afa2b23dbdf7db
#0 34.26       creating build/temp.linux-aarch64-cpython-310/tmp/pip-install-6klh6n1t/cftime_bd2c91e5ac164b95a6afa2b23dbdf7db/src
#0 34.26       creating build/temp.linux-aarch64-cpython-310/tmp/pip-install-6klh6n1t/cftime_bd2c91e5ac164b95a6afa2b23dbdf7db/src/cftime
#0 34.26       gcc -pthread -B /opt/conda/compiler_compat -Wno-unused-result -Wsign-compare -DNDEBUG -fwrapv -O3 -Wall -fPIC -O3 -isystem /opt/conda/include -fPIC -O3 -isystem /opt/conda/include -fPIC -DNPY_NO_DEPRECATED_API=NPY_1_7_API_VERSION -I/tmp/pip-build-env-ovldjjhh/overlay/lib/python3.10/site-packages/numpy/core/include -I/opt/conda/include/python3.10 -c /tmp/pip-install-6klh6n1t/cftime_bd2c91e5ac164b95a6afa2b23dbdf7db/src/cftime/_cftime.c -o build/temp.linux-aarch64-cpython-310/tmp/pip-install-6klh6n1t/cftime_bd2c91e5ac164b95a6afa2b23dbdf7db/src/cftime/_cftime.o
#0 34.26       error: command 'gcc' failed: No such file or directory
#0 34.26       [end of output]
#0 34.26   
#0 34.26   note: This error originates from a subprocess, and is likely not a problem with pip.
#0 34.26   ERROR: Failed building wheel for cftime
#0 34.26 Failed to build cftime
#0 34.26 ERROR: Could not build wheels for cftime, which is required to install pyproject.toml-based projects
------
failed to solve: process "/bin/bash -o pipefail -c pip3 install scipy pandas matplotlib xarray dask[dataframe] netCDF4 nc-time-axis" did not complete successfully: exit code: 1
```

**Comment by ka7eh** [comment]
Fixed in #152
